### PR TITLE
MB-12483 update the FOUO banner to CUI

### PR DESCRIPTION
**Pull request by YanZ777**
## Summary

This does not handle adding the banner to the customer application. This only covers changing the language of the banner and doing a refactor so that all the filenames are consistent.

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the office and admin app
2. There should be a black banner with green text that says "CONTROLLED UNCLASSIFIED INFORMATION"
3. Login and verify that the banner is still there with the correct text

## Verification Steps for Author

These are to be checked by the author.

- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

## Screenshots

image

image

### CODE REVIEW

File: pkg/models/shipment_summary_worksheet.go [hunk 1]
```diff
@@ -350,13 +350,13 @@ func FetchMovingExpensesShipmentSummaryWorksheet(move Move, db *pop.Connection,
 }
 
 const (
-	forOfficialUseOnlyText = "UNCLASSIFIED // FOR OFFICIAL USE ONLY"
+	controlledUnclassifiedInformationText = "Controlled Unclassified Information"
```
Comment (YanZ777) [inline pkg/models/shipment_summary_worksheet.go:353]: I'm not sure if this code is still being used. I've opted to leave it in since I think determining that is out of scope. 
I've change the language here to say "Controlled Unclassified Information"

File: src/components/CUIHeader/CUIHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+
+import styles from './CUIHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+const CUIHeader = () => (
+  <div className={cx('cui-header')}>
+    <div className={cx('cui-header--text')}>Controlled Unclassified Information</div>
```
Comment (YanZ777) [inline src/components/CUIHeader/CUIHeader.jsx:10]: Should this be in caps?
The header will display it in caps because of the CSS, but I'm not sure if I should also just do that here too.

File: pkg/models/shipment_summary_worksheet.go [hunk 3]
```diff
@@ -350,13 +350,13 @@ func FetchMovingExpensesShipmentSummaryWorksheet(move Move, db *pop.Connection,
 }
 
 const (
-	forOfficialUseOnlyText = "UNCLASSIFIED // FOR OFFICIAL USE ONLY"
+	controlledUnclassifiedInformationText = "CONTROLLED UNCLASSIFIED INFORMATION"
```
Comment (YanZ777) [inline pkg/models/shipment_summary_worksheet.go:353]: I'm not sure if this code is still being used. I've opted to leave it in since I think determining that is out of scope.
I've change the language here to say "Controlled Unclassified Information"

File: src/components/CUIHeader/CUIHeader.jsx (hunk 2 above)
Comment (kylehilltruss) [reply src/components/CUIHeader/CUIHeader.jsx:10]: I actually think the title case you used is more correct for a11y purposes, but it's the finest of hairs to split either way

File: src/components/CUIHeader/CUIHeader.module.scss [hunk 4]
```diff
@@ -4,7 +4,7 @@
 @import '../../shared/styles/mixins';
 @import '../../shared/styles/colors';
 
-.fouo-header {
+.cui-header {
```
Comment (kylehilltruss) [inline src/components/CUIHeader/CUIHeader.module.scss:7]: Thank you for changing the class/component names while you were at it!

**Review by kylehilltruss — APPROVED** [review]
Everything looks good and sensible to me.

**Review by AnnaGingle — APPROVED** [review]
Looks good!

**Review by LeDeep — APPROVED** [review]
looks good 🚀
